addons/l10n_sa_edi/tests/common.py

```diff
@@ -195,8 +196,9 @@ def setUpClass(cls, chart_template_ref='l10n_sa.sa_chart_template_standard', edi
                     <InstructionNote>___ignore___</InstructionNote>
                 </xpath>
                 '''
+        cls.user_saudi = new_test_user(cls.env, 'xav', email='em@il.com', notification_type='inbox', groups='account.group_account_invoice', tz='Asia/Riyadh')
 
-    def _create_invoice(self, **kwargs):
+    def _create_invoice(self, saudi_user=False, **kwargs):
```

Comment:
Instead of passing a flag saying that we use a `saudi_user`, what about passing the user in `kwargs` and remove this parameter?
We can check in `kwargs` if we have a specified user and use `with_user` with it.